packages/amplify-e2e-core/src/categories/api.ts

```diff
@@ -919,7 +919,7 @@ export function addApiWithCognitoUserPoolAuthTypeWhenAuthExists(
 export function addRestContainerApi(projectDir: string, opts: Partial<AddApiOptions & { apiKeyExpirationDays: number }> = {}) {
   const options = _.assign(defaultOptions, opts);
   return new Promise<void>((resolve, reject) => {
-    spawn(getCLIPath(), ['add', 'api'], { cwd: projectDir, stripColors: true })
+    spawn(getCLIPath(), ['add', 'api', '--debug'], { cwd: projectDir, stripColors: true })
```

Comment:
why add the debug flag?

Reply:
Not needed, removed in https://github.com/aws-amplify/amplify-category-api/pull/2377